addons/im_livechat/static/src/core/common/thread_patch.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.Thread" t-inherit-mode="extension">
+        <xpath expr="//div[hasclass('o-mail-Thread')]" position="before">
+            <div t-if="showVisitorDisconnected" class="bg-dark-subtle p-2 fw-bold">
+                <i class="fa fa-clock-o mx-1"/> Visitor is disconnected
+            </div>
+        </xpath>
+    </t>
+</templates>
```

Comment:
`bg-secondary` is better than `bg-dark-subtle`:

`bg-dark-subtle`:

![Screenshot 2025-01-24 at 16 44 42](https://github.com/user-attachments/assets/c9300464-7a0e-494a-a956-0b57e20b1671)
![Screenshot 2025-01-24 at 16 44 51](https://github.com/user-attachments/assets/1d354dab-5b11-4234-b7d5-80f2219d49cb)

`bg-secondary`:
![Screenshot 2025-01-24 at 16 44 31](https://github.com/user-attachments/assets/8e1e05f3-c03e-4f5d-9ac9-7df27631a924)
![Screenshot 2025-01-24 at 16 44 21](https://github.com/user-attachments/assets/ededa7cb-9067-4820-96c3-90fe0a327f8b)